src/cluster_legacy.c

```diff
@@ -7040,6 +7086,15 @@ int clusterParseSetSlotCommand(client *c, int *slot_out, clusterNode **node_out,
         return 0;
     }
 
+    if (clusterIsAnySlotImporting()) {
+        addReplyError(c, "A slot is currently being imported via CLUSTER MIGRATESLOTS. Please cancel any ongoing import operations and try again.");
```

Comment:
I don’t think we should embed mitigation steps in error messages. there isn't enough room to provide sufficient context, and they get stale too. It’s better to keep the error factual and move any guidance into docs/best-practice doc. we should also scrub other newly introduced errors for alignment too.

```suggestion
        addReplyError(c, "Slot import in progress.");
```

Reply:
Makes sense. I scanned the rest and fixed